File: alarmclock/src/test/java/com/externalalarmclock/alarmclock/resources/CapabilitiesResourceTest.java

package com.externalalarmclock.alarmclock.resources;

import org.junit.Assert;
import org.junit.Test;

public class CapabilitiesResourceTest {
	@Test
	public void getCapabilitiesTest() {
		CapabilitiesResource res = new CapabilitiesResource();

		Assert.assertEquals(true, res.getCapabilities().isWakeupLight());
	}
}

File: alarmclock/src/test/kotlin/com/externalalarmclock/alarmclock/resources/CapabilitiesResourceTest.kt
package com.externalalarmclock.alarmclock.resources

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class CapabilitiesResourceTest {
    @Test
    fun getCapabilitiesTest() {
        val res = CapabilitiesResource()

        Assertions.assertEquals(true, res.capabilities.isWakeupLight)
    }
}
